### PR TITLE
Creating tutorial edits

### PR DESCRIPTION
**Pull request by bagnacan**
I edited the "create new tutorial" tutorial.
In particular, I made modifications for:
- fixing spelling, syntax, small/upper-cases
- re-wording explanations

### CODE REVIEW

File: topics/training/tutorials/create-new-tutorial-content/tutorial.md [hunk 1]
```diff
@@ -108,21 +115,21 @@ Once we set up the infrastructure, we are ready to write the tutorial
 
 ## Finding a good toy dataset
 
-The first question to come is which data using for the tutorial and particularly for the hands-on parts. This data must be informative enough to illustrate the use of a tool or a technic, but it must not be too big to be able to be run during workshop or locally. Typically, this is a subset of a full dataset where the informative data has been extracted.
+The first question to come is what data to use for waltking the tutorial through the hands-on sections. The selected data must be informative enough to illustrate the meaning of using a tool or a given technique, but not too big to reqire long waiting times for its processing during a workshop. Typically, the selected data should be the informative subset of a full real-life dataset.
```
Comment (bebatut) [inline topics/training/tutorials/create-new-tutorial-content/tutorial.md]: "waltking"?

File: topics/training/tutorials/create-new-tutorial-content/tutorial.md [hunk 2]
```diff
@@ -108,21 +115,21 @@ Once we set up the infrastructure, we are ready to write the tutorial
...
     - Blasting it on the NR database on [NCBI Blast](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome)
     - Extracting one similar sequence found with Blast
-    - Searching and extracting 2 other sequences of the species using the [NCBI Nucleotide database](https://www.ncbi.nlm.nih.gov/nuccore)
+    - Searching and extracting 2 other sequences of the same species using the [NCBI Nucleotide database](https://www.ncbi.nlm.nih.gov/nuccore)
 
-We then developed the tutorial and tested it on this dataset. Once we were ready to share it, we uploaded the datasets on [Zenodo](https://zenodo.org/) to obtain a dedicated DOI (in the [Galaxy training network community](https://zenodo.org/communities/galaxy-training/?page=1&size=20)).
+We then developed the tutorial and tested it on this toy dataset. Once we were ready to share it, we uploaded the toy datasets on [Zenodo](https://zenodo.org/) to obtain a dedicated DOI (in the [Galaxy training network community](https://zenodo.org/communities/galaxy-training/?page=1&size=20)).
 
-## Filling the tutorial content
+## Edit the tutorial
```
Comment (bebatut) [inline topics/training/tutorials/create-new-tutorial-content/tutorial.md]: It is a bit more than just "Edit", I think

File: topics/training/tutorials/create-new-tutorial-content/tutorial.md (hunk 1 above)
Comment (bagnacan) [reply topics/training/tutorials/create-new-tutorial-content/tutorial.md]: ah, right, thanks. It should be "walking"

File: topics/training/tutorials/create-new-tutorial-content/tutorial.md (hunk 2 above)
Comment (bagnacan) [reply topics/training/tutorials/create-new-tutorial-content/tutorial.md]: makes sense

File: topics/training/tutorials/create-new-tutorial-content/tutorial.md [hunk 3]
```diff
@@ -108,21 +115,21 @@ Once we set up the infrastructure, we are ready to write the tutorial
 
 ## Finding a good toy dataset
 
-The first question to come is which data using for the tutorial and particularly for the hands-on parts. This data must be informative enough to illustrate the use of a tool or a technic, but it must not be too big to be able to be run during workshop or locally. Typically, this is a subset of a full dataset where the informative data has been extracted.
+The first question to come is what data to use for walking the tutorial through the hands-on sections. The selected data must be informative enough to illustrate the meaning of using a tool or a given technique, but not too big to reqire long waiting times for its processing during a workshop. Typically, the selected data should be the informative subset of a full real-life dataset.
```
Comment (bgruening) [inline topics/training/tutorials/create-new-tutorial-content/tutorial.md]: require?

**Comment by bgruening** [comment]
Nice, keep it coming! :)

**Pull request merged**